_i18n/en/resources/user-guides/cli_wallet_daemon_isolation_qubes_whonix.md

```diff
@@ -1,25 +1,122 @@
 {% include disclaimer.html translated="no" translationOutdated="no" %}
 
-With [Qubes](https://qubes-os.org) + [Whonix](https://whonix.org) you can have a Monero wallet that is without networking and running on a virtually isolated system from the Monero daemon which has all of its traffic forced over [Tor](https://torproject.org).
+With [Qubes](https://qubes-os.org) + [Whonix](https://whonix.org) you can have a Monero wallet that is without networking and running on a virtually isolated system from the Monero daemon, which has all of its traffic forced over [Tor](https://torproject.org).
 
 Qubes gives the flexibility to easily create separate VMs for different purposes. First you will create a Whonix workstation for the wallet with no networking. Next, another Whonix workstation for the @daemon which will use your Whonix gateway as it's NetVM. For communication between the wallet and daemon you can make use of Qubes [qrexec](https://www.qubes-os.org/doc/qrexec3/).
 
 This is safer than other approaches which route the wallets rpc over a Tor hidden service, or that use physical isolation but still have networking to connect to the daemon. In this way you don't need any network connection on the wallet, you preserve resources of the Tor network, and there is less latency.
 
-## 1. [Create Whonix AppVMs](https://www.whonix.org/wiki/Qubes/Install):
+## 1. Install Monero in template(s)
 
-+ Using a Whonix workstation template, create two workstations as follows:
+The first step is to install Monero daemon and wallet in the template(s).
 
-  - The first workstation will be used for your wallet, it will referred to as `monero-wallet-ws`. You will have `NetVM` set to `none`.
+Monero daemon needs to be installed in a whonix-workstation template, so that all traffic is routed over Tor.
 
-  - The second workstation will be for the `monerod` daemon, it will be referred to as `monerod-ws`. You will have `NetVM` set to the Whonix gateway `sys-whonix`. Before moving on, make sure this workstation has enough private storage. You can estimate how much space you need by checking the size of the [raw blockchain]({{ site.baseurl }}/downloads/#blockchain). Keep in mind that the blockchain will take up more space with time.
+In this guide we will use the official Monero GUI Qt wallet installed in a whonix-workstation template, but any non-custodial wallet, where you can specify what node to use, should work. You can use a different template for your wallet if you want (and it might even be easier to install if the distribution used for the template contains an up-to-date version of your preferred wallet in the repositories).
 
-## 2. In the AppVM `monerod-ws`:
+We will install both the Monero wallet and daemon in the same whonix-workstation template using pre-compiled binaries. You can simplify the installation process by using the `monero` package available in the Whonix repositories, but it is not up-to-date and also only ships CLI wallet.
```

Comment:
Is it safe to use distribution packages? See also: https://github.com/feather-wallet/feather/issues/193#issuecomment-2286984927

Reply:
Should not use the whonix packaged versions

Reply:
The text here doesn't **endorse** using a packaged version of Monero, merely mentions that it is also an option.  

But let me know if you think it shouldn't be mentioned - I'll remove it.

Reply:
It should _not_ be used